src/dao/AbstractContractDAO.js

```diff
@@ -21,6 +21,8 @@ const timestampStart = Date.now()
 let events = []
 
 class AbstractContractDAO {
+  converter = converter
```

Comment:
this is the private property, so underscore and jsdoc _@private_ missed at the start of the name
and let's make it much shorter _converter_ > __u_
_u_ mean _utils_, because not all functions from _converter_ fits converting issues (for instance _isEmptyAddress_)